modules/keyword/keyword.gd

```diff
@@ -0,0 +1,24 @@
+extends Node
+
+
+#region Public Variables
+var keywords: Array[StringName] = []
+#endregion
+
+
+#region Internal Functions
+func _ready() -> void:
+	Modules.register_hooks(handler)
+#endregion
+
+
+#region Public Functions
+func handler(what: StringName, info: Array) -> bool:
+	return true
```

Comment:
This doesn't do anything yet, but it might in the future. Maybe it would be a good idea to not do this until needed?